### PR TITLE
Tests for the gas cost of each opcode

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/GeneralStateTestsFiller/stEIP150singleCodeGasPrices/gasCostExpFiller.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          ; of the parameters
+          [powerOf]       $4
+          [expectedCost]  $36
+
+ 
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; Run the operation
+          [gasB4]    (gas)
+          (exp 2 @powerOf)
+          [gasAfter] (gas)
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stEIP150singleCodeGasPrices/gasCostExpFiller.yml:42]: here the cost of (GAS) opcode itself gona be recorded. 
you can put this as a constant in a contract and to the formula.

**Review by winsvega — APPROVED** [review]
looks very good. I think we can remove all the previous tests in this folder. because yours cover everything

File: src/GeneralStateTestsFiller/stEIP150singleCodeGasPrices/gasCostExpFiller.yml (hunk 1 above)
Comment (qbzzt) [reply src/GeneralStateTestsFiller/stEIP150singleCodeGasPrices/gasCostExpFiller.yml:42]: That is the reason what I check is the total expected cost.

**Comment by qbzzt** [comment]
> looks very good. I think we can remove all the previous tests in this folder. because yours cover everything
> (not in this pr)

Not quite. I covered all the opcodes that were either:

1. Easy

or 

2. Previously uncovered

So we should preserve at least these tests:

0x3C EXTCODECOPY     stEIP150singleCodeGasPrices/RawExtCodeCopyGasFiller.json
0xF1 CALL     stEIP150singleCodeGasPrices/RawCallGasAskFiller.json
0xF0 CREATE     stEIP150singleCodeGasPrices/RawCreateGasFiller.json
0xF2 CALLCODE    stEIP150singleCodeGasPrices/RawCallCodeGasMemoryFiller.json
0xF3 RETURN     stEIP150singleCodeGasPrices/gasCostReturn.yml
0xF4 DELEGATECALL     stEIP150singleCodeGasPrices/RawDelegateCallGasAskFiller.json

**Comment by qbzzt** [comment]
Superceded by PR #745

**Pull request closed without merging**